doc/api-extensions.md

```diff
@@ -926,3 +926,16 @@ Introduces the ability to create a storage pool from an existing non-empty volum
 This option should be used with care, as LXD can then not guarantee that volume name conflicts won't occur
 with non-LXD created volumes in the same volume group.
 This could also potentially lead to LXD deleting a non-LXD volume should name conflicts occur.
+
+## container\_syscall\_intercept\_hugetlbfs
+When mount syscall interception is enabled and hugetlbfs is specified as an
+allowed filesystem type LXD will mount a separate hugetlbfs instance for the
+container with the uid and gid mount options set to the container's root uid
+and gid. This ensure that processes in the container can use hugepages.
+
+##limits\_hugepages.*
```

Comment:
Missing space and invalid name (*) for an extension.

Reply:
Should just be `## limits\_hugepages`

Reply:
fixed